ios/MarkdownTextInputDecoratorView.mm

```diff
@@ -103,11 +94,15 @@ - (void)setMarkdownStyle:(RCTMarkdownStyle *)markdownStyle
   [_markdownUtils setMarkdownStyle:markdownStyle];
 
   // apply new styles
+  if (_textView != nil) {
+    [_textView performSelector:@selector(textDidChange)];
+  } else {
 #ifdef RCT_NEW_ARCH_ENABLED
-  [_textInput _setAttributedString:_backedTextInputView.attributedText];
+    [_textInput _setAttributedString:_backedTextInputView.attributedText];
 #else
-  [_textInput setAttributedText:_textInput.attributedText];
+    [_textInput setAttributedText:_textInput.attributedText];
 #endif /* RCT_NEW_ARCH_ENABLED */
+  }
```

Comment:
Let's add some comments in the code which branch is executed for `UITextView`/`UITextField`.